moby/sys/signal assumes every non-MIPS Linux architecture has
SIGSTKFLT. SPARC does not: it follows the SunOS numbering, where
signal 16 is SIGURG and signal 7 is SIGEMT, so unix.SIGSTKFLT is
undefined there and the package fails to compile. Add a sparc64
variant, mirroring the existing MIPS one. SIGRTMAX is 64, as in the
generic file, so only the signal-map entry differs.

diff -ruN mobya/vendor/github.com/moby/sys/signal/signal_linux.go mobyb/vendor/github.com/moby/sys/signal/signal_linux.go
--- a/vendor/github.com/moby/sys/signal/signal_linux.go	2026-08-23 12:57:39.490700707 +0200
+++ b/vendor/github.com/moby/sys/signal/signal_linux.go	2026-08-23 12:57:39.497145256 +0200
@@ -1,5 +1,5 @@
-//go:build !mips && !mipsle && !mips64 && !mips64le
-// +build !mips,!mipsle,!mips64,!mips64le
+//go:build !mips && !mipsle && !mips64 && !mips64le && !sparc64
+// +build !mips,!mipsle,!mips64,!mips64le,!sparc64
 
 package signal
 
diff -ruN mobya/vendor/github.com/moby/sys/signal/signal_linux_sparc64.go mobyb/vendor/github.com/moby/sys/signal/signal_linux_sparc64.go
--- a/vendor/github.com/moby/sys/signal/signal_linux_sparc64.go	1970-01-01 01:00:00.000000000 +0100
+++ b/vendor/github.com/moby/sys/signal/signal_linux_sparc64.go	2026-08-23 12:57:39.544226748 +0200
@@ -0,0 +1,85 @@
+//go:build linux && sparc64
+// +build linux
+// +build sparc64
+
+package signal
+
+import (
+	"syscall"
+
+	"golang.org/x/sys/unix"
+)
+
+const (
+	sigrtmin = 34
+	sigrtmax = 64
+)
+
+// SignalMap is a map of Linux signals.
+var SignalMap = map[string]syscall.Signal{
+	"ABRT":     unix.SIGABRT,
+	"ALRM":     unix.SIGALRM,
+	"BUS":      unix.SIGBUS,
+	"CHLD":     unix.SIGCHLD,
+	"CLD":      unix.SIGCLD,
+	"CONT":     unix.SIGCONT,
+	"FPE":      unix.SIGFPE,
+	"HUP":      unix.SIGHUP,
+	"ILL":      unix.SIGILL,
+	"INT":      unix.SIGINT,
+	"IO":       unix.SIGIO,
+	"IOT":      unix.SIGIOT,
+	"KILL":     unix.SIGKILL,
+	"PIPE":     unix.SIGPIPE,
+	"POLL":     unix.SIGPOLL,
+	"PROF":     unix.SIGPROF,
+	"PWR":      unix.SIGPWR,
+	"QUIT":     unix.SIGQUIT,
+	"SEGV":     unix.SIGSEGV,
+	"EMT":      unix.SIGEMT,
+	"STOP":     unix.SIGSTOP,
+	"SYS":      unix.SIGSYS,
+	"TERM":     unix.SIGTERM,
+	"TRAP":     unix.SIGTRAP,
+	"TSTP":     unix.SIGTSTP,
+	"TTIN":     unix.SIGTTIN,
+	"TTOU":     unix.SIGTTOU,
+	"URG":      unix.SIGURG,
+	"USR1":     unix.SIGUSR1,
+	"USR2":     unix.SIGUSR2,
+	"VTALRM":   unix.SIGVTALRM,
+	"WINCH":    unix.SIGWINCH,
+	"XCPU":     unix.SIGXCPU,
+	"XFSZ":     unix.SIGXFSZ,
+	"RTMIN":    sigrtmin,
+	"RTMIN+1":  sigrtmin + 1,
+	"RTMIN+2":  sigrtmin + 2,
+	"RTMIN+3":  sigrtmin + 3,
+	"RTMIN+4":  sigrtmin + 4,
+	"RTMIN+5":  sigrtmin + 5,
+	"RTMIN+6":  sigrtmin + 6,
+	"RTMIN+7":  sigrtmin + 7,
+	"RTMIN+8":  sigrtmin + 8,
+	"RTMIN+9":  sigrtmin + 9,
+	"RTMIN+10": sigrtmin + 10,
+	"RTMIN+11": sigrtmin + 11,
+	"RTMIN+12": sigrtmin + 12,
+	"RTMIN+13": sigrtmin + 13,
+	"RTMIN+14": sigrtmin + 14,
+	"RTMIN+15": sigrtmin + 15,
+	"RTMAX-14": sigrtmax - 14,
+	"RTMAX-13": sigrtmax - 13,
+	"RTMAX-12": sigrtmax - 12,
+	"RTMAX-11": sigrtmax - 11,
+	"RTMAX-10": sigrtmax - 10,
+	"RTMAX-9":  sigrtmax - 9,
+	"RTMAX-8":  sigrtmax - 8,
+	"RTMAX-7":  sigrtmax - 7,
+	"RTMAX-6":  sigrtmax - 6,
+	"RTMAX-5":  sigrtmax - 5,
+	"RTMAX-4":  sigrtmax - 4,
+	"RTMAX-3":  sigrtmax - 3,
+	"RTMAX-2":  sigrtmax - 2,
+	"RTMAX-1":  sigrtmax - 1,
+	"RTMAX":    sigrtmax,
+}
